src/components/Trends/ExternalTooltip.jsx

```diff
@@ -43,12 +43,9 @@ export class ExternalTooltip extends React.Component {
       return elements
     }
 
-    elements.push( <span className="u-left a-btn a-btn__link"
+    elements.push( <span className="u-left"
                            id={ 'focus-' + sanitizeHtmlId( value.name ) }
-                           key={ value.name }
-             onClick={ () => {
-               this.props.add( value.name, lens )
```

Comment:
we should remove the add from mapDispatchtoProps, and also remove the tests accordingly since this isn't used anymore.